pkgs/tools/package-management/apx/default.nix

```diff
@@ -30,25 +30,24 @@ buildGoModule rec {
   postInstall = ''
     mkdir -p $out/etc/apx
 
-    cat > "$out/etc/apx/config.json" <<EOF
+    # https://github.com/Vanilla-OS/apx/blob/main/config/apx.json
+    cat > "$out/etc/apx/apx.json" <<EOF
```

Comment:
And I do not think this configuration file is loaded by default: https://github.com/Vanilla-OS/apx/blob/aedd84b64f92fdeb3942d209b26c6f1f2337f335/settings/config.go#L35

Reply:
Many thanks for the useful suggestions! I have incorporated them. But I have an question I would like your input on. Should I bring in `podman` as a dependency, or leave the choice & set-up of containerization up to the user?

`podman` as a dep means it works out of the box. But if the user wants to use docker this overrides that.

Leaving out `podman` as a dep means the user chooses their own containerization set-up. But if they haven't installed `podman` / `docker` it means the app will tell them a core-component is missing. As a point of comparison `distrobox` itself doesn't have either `podman` or `docker` as a dep, though obviously it needs one or both to be installed in order to run. I'm inclined to leave it out following `distrobox`. Any thoughts?

Reply:
Since both docker (requiring a daemon), and podman (requiring some configuration files?) need some system-wide setup to work correctly, I think leaving them out is fine. If an out of the box experience is needed, we can provide a nixos module instead.